src/linear_algebra/Hermitian_inner_product.lean

```diff
@@ -0,0 +1,669 @@
+/-
+Copyright (c) 2018 Andreas Swerdlow. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Andreas Swerdlow
+-/
+
+import analysis.normed_space.basic linear_algebra.sesquilinear_form data.complex.basic
+
+open vector_space field set complex real
+
+set_option class.instance_max_depth 32
+
+lemma ne_comm {α : Type*} {a b : α} : a ≠ b ↔ b ≠ a :=
```

Comment:
This shouldn't be in this file. I think it should go somewhere in `logic.lean` if it isn't already there.

Reply:
Is it worth waiting to moving these all of these lemmas that don't belong until the end of the process, just to keep everything in one place for now?

Reply:
The lemmas are supposed to go in the right place as part of the PR process.

Reply:
It already exists, and is called `ne.symm`.